src/util/codec/mysql/json.rs

```diff
@@ -0,0 +1,696 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Result;
+use std::{str, f64};
+use std::cmp::{Ord, Ordering, PartialEq, PartialOrd};
+use std::collections::BTreeMap;
+use std::result::Result as SResult;
```

Comment:
I hate this kind of `use`. 

since we've already made it tikv coding style.
I'd rather not to say. 😿